tool/build_release.sh

```diff
@@ -63,3 +63,7 @@ flutter build web \
 mv build/web ../devtools/build
 
 popd
+
+pushd packages/devtools
```

Comment:
did you mean to add these lines to build_release?

Reply:
yep. This was a required step to being able to build and run devtools. This happens as part of the publish script but was missing here.